label_studio_ml/tesseract/tesseract.py

```diff
@@ -66,14 +66,19 @@ def predict(self, tasks, **kwargs):
         # extract task metadata: labels, from_name, to_name and other
         from_name, to_name, value = self.label_interface.get_first_tag_occurence("TextArea", "Image")
         task = tasks[0]
-        img_path_url = task["data"][value]
+        print(task["data"][value])
+        pdf_path_url = task["data"][value].split("_")[:-1]
+        page_number = task["data"][value].split("_")[-1]
+        page_number = int(page_number.split(".")[0])
```

Comment:
is there a way to pass PDF filename and page number directly as two separate values?

Reply:
Maybe it's possible. With the current implementation I don't see how. The `task` dictionary is set by label studio. We can probably adjust it somehow.

I think the issue is, that in LabelStudio we work on png files instead of borehole profile level. Therefore the information we're given in the `task` dictionary is on png/page level too.